packages/terra-form-fieldset/src/Fieldset.scss

```diff
@@ -25,6 +25,15 @@
     width: auto;
   }
 
+  .error-text {
+    color: var(--terra-form-fieldset-error-text-color, #e50000);
+    font-size: var(--terra-form-fieldset-error-text-font-size, 0.857rem);
+    font-weight: var(--terra-form-fieldset-error-text-font-weight, normal);
+    line-height: 1.25;
```

Comment:
Should we be making `line-height` themeable for error-text and help-text? @neilpfeiffer

Reply:
Thanks @yuderekyu - currently discussion the options of what fieldset should have/do with @JakeLaCombe